src/main/java/de/hysky/skyblocker/annotations/Init.java

```diff
@@ -0,0 +1,8 @@
+package de.hysky.skyblocker.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+@Target(ElementType.METHOD)
+public @interface Init {
```

Comment:
might be useful to explicitly set the retention policy of the annotation, ideally to `CLASS`